aea/skills/base.py

```diff
@@ -897,6 +779,74 @@ def from_config(
         return skill
 
 
+def _parse_module(
```

Comment:
Good that this is extracted! Let's have some tests +/- too :)

Reply:
yes